src/plugins/sync-plugin.js

```diff
@@ -695,6 +695,17 @@ const createNodeFromYElement = (
     mapping.set(el, node)
     return node
   } catch (e) {
+    try {
+      const attrs = el.getAttributes(snapshot)
+      console.error('[@gamma-app/y-prosemirror][sync-plugin] createNodeFromYElement error:', {
+        message: e.message,
+        el: JSON.stringify(el),
```

Comment:
Are you sure el can be stringified? I'm fine just trying it out but I think it may throw your next error from this

Reply:
I'm not - Lemme verify real quick

Reply:
<img width="1174" alt="Screenshot 2024-04-08 at 2 24 32 PM" src="https://github.com/gamma-app/y-prosemirror/assets/3718812/dfedae6f-3a60-4e85-9b41-39484a9b85ea">